proxy/pkg/config/config.go

```diff
@@ -106,6 +106,7 @@ type Config struct {
 	PreSignedURL          PreSignedURL
 	AutoprovisionAccounts bool
 	EnableBasicAuth       bool
+	Insecure              bool
```

Comment:
Naming: How about rather calling that `SkipCertValidation` or so? 

But this is more in the area of nitpicking - just as you want.

Reply:
It is called Insecure everywhere in oCIS:
- PROXY_OIDC_INSECURE
- KONNECTD_INSECURE
- GLAUTH_BACKEND_INSECURE
- STORAGE_OIDC_INSECURE

Reply:
InsecureBackends to signify what is insecure